app/components/ui/files/publish-modal/component.js

```diff
@@ -173,8 +213,8 @@ export default Ember.Component.extend({
         Responsible for opening the login dialog for the user. Ideally, we could
         tell when the user finishes logging in so that we know when to fetch the token
         */
-       let url = 'https://cn.dataone.org/portal/oauth?action=start&target='+ENV.authRedirect;
-        let newwindow=window.open(url,'auth','height=400,width=450');
+       let url = 'https://cn-stage-2.test.dataone.org/portal/oauth?action=start&target='+config.authRedirect;
```

Comment:
Does the Dashboard have any central configuration store that this URL could be factored out into?

Reply:
I can stick this in `environment.js`

Reply:
👍

Reply:
Addressed in 3e8b9afb676cac4146bc21e1330f77616c15db60